exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/dml/ConditionsTests.kt

```diff
@@ -378,4 +378,18 @@ class ConditionsTests : DatabaseTestsBase() {
             )
         }
     }
+
+    @Test
+    fun testNullOrEmpty() {
+        val tester = object : IntIdTable("tester") {
+            val name = text("name").nullable()
+        }
+
+        withTables(tester) {
+            tester.insert { it[name] = null }
+            tester.insert { it[name] = "" }
```

Comment:
For the testing purposes it could be good to add an entry with non empty `name`. What if condition simply skipped and all the rows returned.